sycl/test-e2e/Basic/interop/make_kernel_subdevice_l0.cpp

```diff
@@ -0,0 +1,136 @@
+// REQUIRES: level_zero, opencl, level_zero_dev_kit
+// RUN: %{build} %level_zero_options -lOpenCL -o %t.ze.out
+// RUN: env ONEAPI_DEVICE_SELECTOR="level_zero:*" %t.ze.out
+
+#include <CL/sycl.hpp>
```

Comment:
This has been deprecated for more than a year. Please change to `<sycl/sycl.hpp>`.

Reply:
Likewise.